django/api/views.py

```diff
@@ -206,6 +206,15 @@ class ApplicationInstanceViewSet(viewsets.ModelViewSet):
     queryset = ApplicationInstance.objects.order_by('version_code')
     serializer_class = ApplicationInstanceSerializer
 
+    @detail_route(methods=['GET'], permission_classes=[permissions.AllowAny, ])
```

Comment:
Same as in "download": Users that are not logged in should not be able to fetch apps.

Reply:
yes I know. But since there's no captcha mechanism, it's still possible to expose this entry

Reply:
although this can be easily patched